src/index.html

```diff
@@ -8,6 +8,11 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0"></div>
```

Comment:
add 5 divs in each div and use them for stars
Code structure should look like this
```
<div class="stars stars--0">
  <div></div>
  <div></div>
  <div></div>
  <div></div>
  <div></div>  
</div>

<div class="stars stars--1">
  <div></div>
  <div></div>
  <div></div>
  <div></div>
  <div></div>  
</div>
...
```